Src/AutoFakeItEasy/Properties/AssemblyInfo.cs

```diff
@@ -33,8 +33,9 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("3.50.2.0")]
-[assembly: AssemblyFileVersion("3.50.2.0")]
+[assembly: AssemblyVersion("1.0.0.0")]
+[assembly: AssemblyFileVersion("1.0.0.0")]
+[assembly: AssemblyInformationalVersion("1.0.0.0")]
```

Comment:
Hmm... Wouldn't this be a little confusing to those trying to compile AutoFixture directly from source code?